index.html

```diff
@@ -72,6 +78,7 @@ <h3 class="title flex">I'am a Software Developer</h3>
       <h2 class="work-header">My recent Works</h2>
```

Comment:
- [ ] OPTIONAL: This header is aligned left and also followed by a horizontal line to the end in the original design. Kindly compare yours and the original design from the screenshot images below. 
![image](https://user-images.githubusercontent.com/64894927/121423685-34dc2300-c979-11eb-916c-ccf4d4bf989d.png)    
![image](https://user-images.githubusercontent.com/64894927/121423722-402f4e80-c979-11eb-8ca6-d4c131ee96ff.png)